components/server/src/server.rs

```diff
@@ -1092,18 +1090,35 @@ impl<ER: RaftEngine> TiKVServer<ER> {
 
                 let mut available = capacity.checked_sub(used_size).unwrap_or_default();
                 available = cmp::min(available, disk_stats.available_space());
-                if available <= disk_reserved {
+                let disk_status = disk::get_disk_status(0);
```

Comment:
How about `prev_disk_status` and `curr_disk_status`, and change log to `warn!("disk usage {} -> {}, ...")`?